tests/admin/assets-operations.test.ts

```diff
@@ -33,19 +32,23 @@ describe('Coinflip admin assets entrypoints test', function () {
   let emptyCoinflips: Record<string, Coinflip> = {};
 
   beforeAll(async () => {
-    const fa2Wrappers = await makeFA2();
-    const fa2TokenAddress = fa2Wrappers.alice.contract.address;
-    testFA2TokenAsset = {
-      fa2: {
-        address: fa2TokenAddress,
-        id: new BigNumber(defaultFA2TokenId)
-      }
-    };
-    emptyCoinflips = await makeEmptyCoinflip();
-    allAssetsAddedCoinflips = await makeAllAssetsAddedCoinflip(
-      fa2TokenAddress,
-      defaultFA2TokenId
-    );
+    try {
```

Comment:
If the beforeAll case can't be executed, the test should fail

Reply:
The necessary exception is thrown here: https://github.com/madfish-solutions/quipuswap-coinflip/pull/6/files#diff-dbf907554f3bfc8a1de8c07acd951b9a6a8d48b2122363de05f34d6b42df2854R51